### ISSUE TITLE
[Android] showCastDialog() not responsive

### ISSUE THREAD

**Issue by YassineDev7** [react-native-google-cast/react-native-google-cast#446]
"react": "18.0.0",
"react-native": "^0.69.3",
"react-native-google-cast": "^4.3.0"

I've got an issue with the cast button on my application 
When i start the application and press the cast button , i used a handler in order to call the function `GoogleCast.showcastDialog()`
the cast window show up but keep searching a device without any return
<img width="415" alt="2022-08-05_01-25-54" src="https://user-images.githubusercontent.com/57623317/182977889-6e6a7edb-0748-42de-8e6d-759c2fc4bb05.png">

**Comment by petrbela** [comment]
Hi! Does the same thing happen if you press the Cast button instead? Is there anything in the debug panel in Android Studio? Have you checked [Troubleshooting](https://react-native-google-cast.github.io/docs/getting-started/troubleshooting)?

**Comment by YassineDev7** [comment]
yeah i already checked Troubleshooting, and also nothing on debug panel. this is my code it will be helpful for u:
`

import React from 'react';
import {
  Button,
  View
} from 'react-native';
import GoogleCast from 'react-native-google-cast';
import { CastButton, useRemoteMediaClient } from 'react-native-google-cast'

import Orientation from 'react-native-orientation';

import styles from './src/assets/css/main.style.js';

const App = () => {
  Orientation.lockToLandscape();
  async function handleCast() {
    try {
      GoogleCast.showCastDialog();
    } catch (error) {
      console.log(error)
    }
  }
 

  const client = useRemoteMediaClient()

  if (client) {
    // Send the media to your Cast device as soon as we connect to a device
    // (though you'll probably want to call this later once user clicks on a video or something)
    client.loadMedia({
      mediaInfo: {
        contentUrl:
          'https://commondatastorage.googleapis.com/gtv-videos-bucket/CastVideos/mp4/BigBuckBunny.mp4',
      },
    })
  }

  return (

    <View style={styles.fullPage}>
      <CastButton style={{ tintColor: 'transparent' }} />
      <Button onPress={handleCast} title="Connect tv" size={28} />
    </View>

  );
};



export default App;
`

**Comment by petrbela** [comment]
Well it seems no device can be found. Which part of Troubleshooting did work and which didn't? Can you see the Cast device from other apps? If so, which? What about the example Cast app?

**Comment by petrbela** [comment]
And when you say "not responsive", does it mean the app freezes? Or that it just keeps searching?

**Comment by YassineDev7** [comment]
it keeps searching without any return, i try the cast from chrome browser as it explain on Troubleshooting and i can find the device and do the cast without problem.

**Comment by petrbela** [comment]
@YassineDev7 Could you set up a new project from scratch (e.g. https://github.com/react-native-google-cast/RNGCExpo) and see if it works there?

**Comment by YassineDev7** [comment]
Hi, it works thanks, i just want to hide the No Media Selected...  popup so I don't like to have it when the cast starts.
